pushapkscript/test/helpers/task_generator.py

```diff
@@ -36,6 +38,11 @@ def generate_json(self):
               "taskId": "PKP2v4y0RdqOuLCqhevD2A",
               "taskType": "signing"
             }}],
-            "google_play_track": "alpha"
+            "google_play_track": "{}"
```

Comment:
Would it make sense to

`'''..."google_play_track": "{track}"'''.format(track=self.google_play_track)`

just to be more explicit?  I'm fine with a single `{}` but when there are 3 across multiple lines, it gets a little more dicey.